Documentation/admin.rst

```diff
@@ -155,8 +155,10 @@ Here is an example Daemon Set definition:
 	        args:
 	          - "-t"
 	          - "vxlan"
-	          - "--etcd-config-path"
-	          - "/var/lib/cilium/etcd-config.yml"
+	          - "--kvstore"
```

Comment:
`with-cni.sh` was the old way of installing as a CNI plugin. It is no longer needed. The master branch is now using `cni-install.sh`.